ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html

```diff
@@ -275,6 +275,10 @@
         <td class="dataset-details">{{ pkg_dict.privacy_contains_pii}} </td>
         </tr>
         {% endif %}
+        <tr>
+        <th scope="row" class="dataset-label">{{ _("privacy: contains SSN") }} </th>
+        <td class="dataset-details">{{ pkg_dict.privacy_contains_ssn|default('no')}} </td>
```

Comment:
@mattboehm The default above was "No", shouldn't it be the same here?

Reply:
@mattboehm Does this change need to be made?  If not I will just merge, thanks!

Reply:
@tanderegg Sorry, wanted to look into this further before responding. Unfortunately, all these yes/no fields have inconsistent casing in the CSV export, so we may want a separate ticket to fix that (probably due to edits made at different times or DIG import vs manual setting). The lowercase "no" is consistent with how other fields are displayed on this page so if anything, I'd change the capitalized one, but for the time being, I think we could merge this as-is.
![screenshot of export](https://user-images.githubusercontent.com/1254555/34569941-4bad9b30-f138-11e7-9c93-11c0b0b61e50.PNG)

Reply:
Actually, lowercased it in the export as this is marginally more consistent.

Reply:
Agreed, can you create an issue to track this in Github?  Will merge now.